3.3/.sti/bin/assemble

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+source scl_source enable python33
+set -e
+
+echo "---> Installing application source"
+cp -Rf /tmp/src/* ./
+
+echo "---> Building your Python application from source"
+if [ -f setup.py ]; then
+    python setup.py install --user
+elif [ -f requirements.txt ]; then
```

Comment:
Yes, and that's usually the case where `requirements.txt` are read into `setup.py`. But sometimes people want to have two different requirement sets: one for test and the other for running. This should address this.